packages/map-template/src/components/Wayfinding/Wayfinding.jsx

```diff
@@ -92,6 +93,12 @@ function Wayfinding({ onStartDirections, onBack, directionsToLocation, direction
 
     const [travelMode, setTravelMode] = useRecoilState(travelModeState);
 
+    /** Indicate if the user has My Position selected */
+    const [myPositionSelected, setMyPositionSelected] = useState(false);
+
+     /** Indicate if the option to choose My Position should be shown */
+    const [showMyPosition, setShowMyPosition] = useState(true);
```

Comment:
This can also indicate that the position is shown. 
It's just a button to tell the directions to use that position, right?

Reply:
We need a way to distinguish between the state when the My Position option has been selected and when the My Position option should be shown. I tried to merge those to into one single state, but could not get it to work. 

We use the `showMyPosition` because we would like to show the option even when a search has not been yet performed. So therefore we use it when the search field has been clicked, the my position option should be shown, and we cannot set the `myPositionSelected` state in that case. 

The same applies when a location has been clicked, we need to use the `showMyPosition` in that case, otherwise we get into a state where we have both the My Position option and the details for the selected location.

And we should also use the `showMyPosition` when there are no results. 

Let me know if it doesn't make sense or if you have any other suggestion of how to approach this. 😄

Reply:
Naming is hard! Could it be `showMyPositionOption`?

Reply:
Yeah, good idea actually! I changed the name.